sami2py/_core_class.py

```diff
@@ -68,7 +68,7 @@ def __init__(self, tag, lon, year, day, outn=False, test=False):
         deni : (4D ndarray)
             Ion density by species (cm^-3)
```

Comment:
```suggestion
            Ion density for each ion species (cm^-3)
```

Reply:
Sounds good, should I just go ahead and make sure all of the SPASE/PSL netCDF standard attrs are there?

Reply:
I'll leave it up to you depending on how much work that takes.